apps/io-wallet-support-func/local.settings.json.example

```diff
@@ -5,6 +5,7 @@
     "CosmosDbEndpoint": "COSMOS_DB_ENDPOINT",
     "CosmosDbDatabaseName": "COSMOS_DB_DATABASE_NAME",
     "CosmosDbRequestTimeout": "COSMOS_DB_REQUEST_TIMEOUT",
-    "HttpRequestTimeout": "HTTP_REQUEST_TIMEOUT"
+    "HttpRequestTimeout": "HTTP_REQUEST_TIMEOUT",
+    "APPLICATIONINSIGHTS_CONNECTION_STRING": "APPLICATIONINSIGHTS_CONNECTION_STRING"
```

Comment:
can't you use managed identities? CC @Krusty93

Reply:
as far as I know the best way is through the SDK and connection string